specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,361 @@
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Expand"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/NetworkData"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkDataList": {
+      "type": "object",
+      "description": "List of network data",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkData"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "NetworkData": {
+      "type": "object",
+      "description": "Network data on a resource",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkDataProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "NetworkDataProperties": {
+      "type": "object",
+      "description": "Describes the properties of the network data.",
+      "properties": {
+        "connectableResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkDataConnectableResource"
+          }
+        }
+      }
+    },
+    "NetworkDataConnectableResource": {
+      "type": "object",
+      "description": "Describes the allowed inbound and outbound traffic of an Azure resource",
+      "properties": {
+        "connectableResourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The connectable resource id"
+        },
+        "connectionType": {
+          "type": "string",
+          "description": "The connection type between the extension resource to the connectable resource Id",
+          "enum": [
+            "Azure vm"
+          ],
+          "x-ms-enum": {
+            "name": "connectionTypeValues",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Azure vm"
+              }
+            ]
+          }
+        },
+        "allowedConnections": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The allowed connections details",
```

Comment:
>The allowed connections details [](start = 26, length = 31)

It isn't clear from the name/description whether this means the connections that were allowed into/out of the resource or if it represents some secure set of connections the user intended to "allow" (i.e. via NSG).

Reply:
This is "the connections that were allowed into/out of the resource". All descriptions here (in all ASC Rest actually) are under PM/technical writer review in order to make them clearer to users. However, we need this API in the manifest for private preview so once the description is updated we issue a specific PR with description changes only.